docs/IE_DG/network_state_intro.md

```diff
@@ -204,10 +215,12 @@ Decsriptions can be found in [Samples Overview](./Samples_Overview.md)
 
 If the original framework does not have a special API for working with states, after importing the model, OpenVINO representation will not contain Assign/ReadValue layers. For example, if the original ONNX model contains RNN operations, IR will contain TensorIterator operations and the values will be obtained only after the execution of whole TensorIterator primitive, intermediate values from each iteration will not be available. To be able to work with these intermediate values of each iteration and receive them with a low latency after each infer request, a special LowLatency transformation was introduced.
 
-LowLatency transformation changes the structure of the network containing [TensorIterator](../ops/infrastructure/TensorIterator_1.md) by adding the ability to work with state, inserting Assign/ReadValue layers as it is shown in the picture below.
+LowLatency transformation changes the structure of the network containing [TensorIterator](../ops/infrastructure/TensorIterator_1.md) and [Loop](../ops/infrastructure/Loop_5.md) by adding the ability to work with state, inserting Assign/ReadValue layers as it is shown in the picture below.
 
 ![applying_low_latency_example](./img/applying_low_latency.png)
 
+Note: After applying the transformation, ReadValue operations can receive other operations as input as shown in the pic above. These inputs should set the initial value for initialization of ReadValue ops, however, such initialization is not supported in the current State API implementation. Input values will be ignored and initialization will be zeros unless otherwise specified by the user via [State API](#openvino-state-api).
```

Comment:
```suggestion
Note: After applying the transformation, ReadValue operations can receive other operations as an input, as shown in the picture above. These inputs should set the initial value for initialization of ReadValue operations. However, such initialization is not supported in the current State API implementation. Input values are ignored and initialization will be zeros unless otherwise specified by the user via [State API](#openvino-state-api).
```